core/storage/skill/gae_models.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Models for storing the skill data models."""
+
+from core.platform import models
+
+from google.appengine.ext import ndb
+
+(base_models,) = models.Registry.import_models([models.NAMES.base_model])
+
+
+class SkillSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
+    """Storage model for the metadata for a skill snapshot."""
+    pass
+
+
+class SkillSnapshotContentModel(base_models.BaseSnapshotContentModel):
+    """Storage model for the content of a skill snapshot."""
+    pass
+
+
+class SkillModel(base_models.VersionedModel):
+    """Model for storing Skills.
+
+    This class should only be imported by the skill domain file, the
+    skill services file, and the skill model test file.
+    """
+    SNAPSHOT_METADATA_CLASS = SkillSnapshotMetadataModel
+    SNAPSHOT_CONTENT_CLASS = SkillSnapshotContentModel
+    ALLOW_REVERT = True
```

Comment:
Actually, maybe this should be False. Skill editing is controlled so I don't think we need to support reverts yet.

Reply:
Done.